examples/components/MNIST/traininsilo/run.py

```diff
@@ -289,7 +350,12 @@ def get_arg_parser(parser=None):
     parser.add_argument(
         "--iteration_num", type=int, required=False, help="Iteration number"
     )
-
+    parser.add_argument(
```

Comment:
nit: similarly, might be slightly better to move this "total_num_of_iterations" to those dp args section, since this arg is only for DP purpose.